newrelic_lambda_cli/subscriptions.py

```diff
@@ -83,10 +85,10 @@ def _remove_subscription_filter(session, function_name, filter_name):
 @catch_boto_errors
 def create_log_subscription(input, function_name):
     assert isinstance(input, SubscriptionInstall)
```

Comment:
Again here, we might consider adding a message about old log ingestion functions.
```suggestion
    assert isinstance(input, SubscriptionInstall)
    function = get_function(input.session, "newrelic-log-ingestion")
    if function:
        warning("It looks like an old log ingestion function is present in this region. Consider manually deleting this as it is no longer used and has been replaced by a log ingestion function specific to the stack.")
```